MekHQ/src/mekhq/gui/view/PersonViewPanel.java

```diff
@@ -396,6 +404,52 @@ private void fillStats() {
         gridBagConstraints.anchor = GridBagConstraints.NORTHWEST;
         pnlStats.add(lblStatus2, gridBagConstraints);
 
+        if (campaign.getCampaignOptions().getUseTimeInService()) {
+            if (null != person.getRecruitmentAsString() && !person.isDependent() && !person.isPrisoner() && !person.isBondsman()) {
```

Comment:
Should be:
`(null != person.getRecruitmentAsString())`

Reply:
ok, easy fix